fuzz/fuzz_targets/fuzz_target_1.rs

```diff
@@ -17,10 +17,10 @@ where
 {
     let mut writer = Writer::new(Cursor::new(Vec::new()));
     let mut buf = vec![];
-    let reader = reader
-        .expand_empty_elements(true)
-        .trim_text(true)
-        .trim_text_end(true);
+    let config = reader.config_mut();
+    config.expand_empty_elements = true;
+    config.trim_text(true);
+    config.trim_text_end = true;
```

Comment:
I slightly dislike the aesthetics here and inability to chain (because of converting methods to members), but ultimately it's an incredibly minor detail